apps/api/src/server/applications/s51advice/s51-advice.routes.js

```diff
@@ -26,4 +26,24 @@ router.post(
 	asyncHandler(createS51Advice)
 );
 
+router.get(
+	'/:id',
+	/*
+        #swagger.tags = ['Applications']
+        #swagger.path = '/applications/s51-advice/{id}'
```

Comment:
I wonder if the application id should also be in the path - it pretty much is for all the other endpoints.  so /applications/{caseId}/s51-advice/{id} ?